packages/server/src/trpc-routers/__tests_e2e__/swap-router.test.ts

```diff
@@ -160,29 +153,6 @@ function assertValidQuote({
 
   // timeMs
   expect(isNumeric(quote.timeMs)).toBeTruthy();
-
-  // Token in fee amount fiat value
-  expect(quote.tokenInFeeAmountFiatValue).toBeInstanceOf(PricePretty);
```

Comment:
Note to reviewer: these fields were removed to reduce the number of queries and caches. Instead, we find workarounds from other data fields in the `useSwap` hook